client/src/components/Draftail/Draftail.scss

```diff
@@ -114,8 +122,87 @@ $draftail-editor-font-family: $font-sans;
   }
 }
 
+.Draftail-BlockToolbar {
```

Comment:
I think there’s now enough in there that I should create separate files. If that’s ok with you I’ll only do this once the RC is out.